### ISSUE TITLE
No module named 'passageidentity.openapi_client'

### ISSUE THREAD

**Issue by bardt** [passageidentity/passage-python#52]
After installing `passage-identity`, I run `main.py` with a single line `from passageidentity import Passage` in it. I get this error:

```
Traceback (most recent call last):
  File "/Volumes/web/collection/main.py", line 1, in <module>
    from passageidentity import Passage
  File "/opt/homebrew/lib/python3.11/site-packages/passageidentity/__init__.py", line 2, in <module>
    from .passage import Passage
  File "/opt/homebrew/lib/python3.11/site-packages/passageidentity/passage.py", line 18, in <module>
    from .openapi_client.api import AppsApi, MagicLinksApi, TokensApi, UsersApi, UserDevicesApi
ModuleNotFoundError: No module named 'passageidentity.openapi_client'
```

**Comment by ctran88** [comment]
Hi @bardt this should be fixed now in [v2.3.1](https://pypi.org/project/passage-identity/2.3.1/) with https://github.com/passageidentity/passage-python/pull/53

**Comment by bardt** [comment]
Confirmed, it's fixed. Thank you!
